apps/pie-microsite/src/_11ty/shortcodes/tokensTable/handleTokenData.js

```diff
@@ -1,6 +1,21 @@
 const pieTokensMetadata = require('../../../tokensMetadata.json');
-const pieTokenCategories = require('../../../tokenCategories.json');
 const { objectHelpers } = require('../../../utilities/helpers');
+const tokenTypes = require('../../../_data/tokenTypes');
+
+/**
+ * Gets the correct column size for each example based on token type.
+ * @param {string} tokenType - - the type of token i.e. color, spacing, radius
+ * @returns {string} - css variable containing correct column size
+ */
+const getExampleColumnSize = tokenType => {
+    const tokenColumnHandler = {
+        [tokenTypes.COLOR]: '240px',
+        [tokenTypes.FONT]: '240px',
+        default: '152px'
+    };
+
+    return `--template-columns: ${(tokenColumnHandler[tokenType] || tokenColumnHandler.default)}`;
+};
```

Comment:
this might not be the place for this function - happy to move it elsewhere, maybe to utilities? thought it was better extracting than repeating in `tokensTable/index.js`